dev/VSIX/Directory.Build.props

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
 
-    <!-- Provide default nuget feed (Reunion internal) and package versions for developer builds -->
+    <!-- Provide default nuget feed (Windows App SDK internal) and package versions for developer builds -->
     <PropertyGroup>
         <RestoreSources Condition="'$(RestoreSources)'==''">
             https://pkgs.dev.azure.com/microsoft/ProjectReunion/_packaging/Project.Reunion.nuget.internal/nuget/v3/index.json
         </RestoreSources>
         <CppWinRTVersion Condition="'$(CppWinRTVersion)' == ''">2.0.210211.2</CppWinRTVersion>
-        <ReunionVersion Condition="'$(ReunionVersion)' == ''">0.5.7</ReunionVersion>
+        <WindowsAppSdkVersion Condition="'$(WindowsAppSdkVersion)' == ''">0.5.7</WindowsAppSdkVersion>
```

Comment:
>0.5.7 [](http://example.com/codeflow?start=74&length=5)

may need to bump the default version

Reply:
It wasn't bumped for 0.8 and it's only used for inner-loop development it's not a rename issue.

Reply:
not critical. but after rename, the default is broken